docs/samples/modelmesh_sdk_sample.ipynb

```diff
@@ -0,0 +1,562 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sample for ModelMesh Serving using the KServe Python SDK\n",
+    "\n",
+    "## This notebook shows how to use the KServe SDK to create, update, and delete an InferenceService\n",
```

Comment:
I don't think we need a subtitle here; normal text should be fine, if we have a descriptive title / page heading we don't need a subtitle